README.md

```diff
@@ -186,8 +186,44 @@ This describes the different sync modes available for the substrate chain and th
 | warp               | partially compatible       | compatible                                                                         |
 | fast / fast unsafe | not compatible             | compatible (with [warnings](https://github.com/paritytech/polkadot-sdk/issues/19)) |
 
+## Forking Kate/Biryani/Dymension
```

Comment:
Do we want to expose the Biryani/Dymension networks as well? @markopoloparadox @Leouarz

Reply:
Not sure, but there won't be validators nor funds for user to use the network, so it's ok I guess.

Reply:
@ToufeeqP Will ask Jakub what he thinks